velox/common/base/tests/BitUtilTest.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/BitUtil.h"
+#include <gtest/gtest.h>
+
+namespace facebook {
+namespace velox {
+namespace test {
+
+TEST(BitUtilTest, forEachBit) {
```

Comment:
There is an existing test for bits manipulating functions in velox/common/tests/BitUtilTest.cpp . It would be nice to move this test there.

Reply:
You are right, I did not see any tests under velox/common/base, where bitUtilTest is and presumed there are no tests .